tilecloud_chain/__init__.py

```diff
@@ -743,7 +743,7 @@ def do(tile):
                     self.log_tiles_error(tile=tile, message=repr(tile.error))
                 return tile
             self.imap(do)
-        if self.config['generation']['maxconsecutive_errors'] > 0:
+        if self.config['generation']['maxconsecutive_errors'] > 0 and role != 'slave':
```

Comment:
Shouldn't be on slave but in demon (and slave?)...

Reply:
It's on a slave that we don't want the process to stop. So we enable MaximumConsecutiveErrors only if we don't run as slave.

Reply:
We should be based on the `--daemon` argument :-)